src/llmcompressor/core/lifecycle.py

```diff
@@ -33,7 +33,7 @@ class CompressionLifecycle:
 
     state: State = field(default_factory=State)
     recipe: Recipe = field(default_factory=Recipe)
-    modifiers: List[StageModifiers] = field(default_factory=list)
+    modifiers: List[Modifier] = field(default_factory=list)
```

Comment:
I'm a little confused, do modifiers exist on the session or do they exist on the recipe? Right now we have both, it'd be nice to focus on one source of truth